csharp/src/Drivers/BigQuery/BigQueryStatement.cs

```diff
@@ -112,7 +112,13 @@ public override QueryResult ExecuteQuery()
             ReadSession rrs = readClient.CreateReadSession("projects/" + results.TableReference.ProjectId, rs, maxStreamCount);
 
             long totalRows = results.TotalRows == null ? -1L : (long)results.TotalRows.Value;
-            IArrowArrayStream stream = new MultiArrowReader(TranslateSchema(results.Schema), rrs.Streams.Select(s => ReadChunk(readClient, s.Name)));
+
+            var readers = rrs.Streams
```

Comment:
To make up for the internal contract change of allowing null on ReadChunk, only pass valid readers (that aren't null) to the MultiArrowReader. If it is empty, then no errors occurs.